src/components/ScoreResult/styled.js

```diff
@@ -0,0 +1,80 @@
+import styled, { css, keyframes } from 'styled-components';
+
+const successGlow = keyframes`
```

Comment:
Personally I kind of like the look of limegreen which is #32CD32
It might be to much. What do you think?

Reply:
Looks a lot better!